Resources/Prototypes/Entities/Objects/Specific/Hydroponics/sprays.yml

```
@@ -14,10 +14,10 @@
   - type: SolutionContainerManager
     solutions:
       spray:
-        maxVol: 100
+        maxVol: 120 # Carpmosia-edit - extendedReagentStorage
```

This is set on the parent.
```suggestion
        #        maxVol: 120 # Carpmosia-edit - extendedReagentStorage
```